qa-tests-backend/src/test/groovy/TLSChallengeTest.groovy

```diff
@@ -61,6 +61,9 @@ class TLSChallengeTest extends BaseSpecification {
         orchestrator.restartPodByLabelWithExecKill("stackrox", [app: "central"])
         orchestrator.waitForPodsReady("stackrox", [app: "central"], 1, 50, 3)
 
+        // Ensure Central API is reachable.
+        withRetry(30, 2) { Services.getMetadataClient().getMetadata() }
```

Comment:
Metadata is now authenticated, maybe ping is a better choice? That's also what Sensors do today for reachability check.